parsers/IL.py

```diff
@@ -71,7 +83,7 @@ def fetch_production(
         "datetime": arrow.now(TZ).datetime,
         "production": {"unknown": production[0] + production[1]},
         "source": IEC_URL,
-        "price": PRICE,
+        "price": price,
```

Comment:
This key is actually ignored.
The data should be returned in the `fetch_price` method, which should have the signature depicted here:
https://github.com/tmrowco/electricitymap-contrib/blob/master/parsers/example.py#L110-L111

Reply:
Oh didn't knew that. will fix.